osu.Game/Screens/Select/BeatmapCarousel.cs

```diff
@@ -375,6 +380,14 @@ private void removeGroup(BeatmapGroup group)
             if (group == null)
                 return;
 
+            if (selectedGroup == group)

```

Comment:
any reason for moving this up in the function?

Reply:
`SelectNext()` doesn't work correctly in the first situation. It can't find selected group in the List of groups because we have already deleted it in Line №378.
And it makes more sence isn't it? Switching to the next beatmap and then deleting previously selected even feels safer then deleting the selected one.

Reply:
just wanted to make sure this was an intentional move, sounds fine.